### PR TITLE
Add local development setup role

### PR DESCRIPTION
**Pull request by greatislander**
This role:

* Network activates Pressbooks.
* Network enables core themes.
* Enables book (blog) registration by logged-in users.
* Enables the plugin menu at the book level.
* Updates the network title to “Pressbooks”.
* Creates a database for running PHPUnit tests.
* Installs the PHPUnit testing framework for Pressbooks.

### CODE REVIEW

**Comment by greatislander** [comment]
@nathanielks I'm particularly looking for your help with [this](https://github.com/pressbooks/trellis/blob/6ebb5f8d2e6a293046af62f77255b8903772f5ba/roles/local-development/tasks/main.yml#L11-L17). I think nested `with_dicts` properties will work based on my understanding of Ansible but if you can give it a quick look and suggest any changes that would be greatly appreciated 👍

File: roles/local-development/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  with_dict: "{{ wordpress_sites }}"
+- name: List themes
+  command: wp theme list --allow-root
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/{{ item.value.current_path | default('current') }}/"
+  with_dict: "{{ wordpress_sites }}"
+- name: Enable default themes
+  command: wp theme enable {{ item[0] }} --network --allow-root
+  args:
+    chdir: "{{ www_root }}/{{ item[1].key }}/{{ item[0].value.current_path | default('current') }}/"
```
Comment (nathanielks) [inline roles/local-development/tasks/main.yml]: needs to be `item[1].value.current_path`

File: roles/local-development/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  with_dict:
```
Comment (nathanielks) [inline roles/local-development/tasks/main.yml]: change to `with_nested`

File: roles/local-development/tasks/main.yml (hunk 1 above)
Comment (greatislander) [inline roles/local-development/tasks/main.yml]: Good catch.

File: roles/local-development/tasks/main.yml (hunk 2 above)
Comment (greatislander) [inline roles/local-development/tasks/main.yml]: Ack, didn't work after all… using a shortcut for now.

**Pull request merged**